ogr/ogrsf_frmts/mysql/ogrmysqldriver.cpp

```diff
@@ -144,6 +144,9 @@ void RegisterOGRMySQL()
 
     poDriver->SetDescription( "MySQL" );
     poDriver->SetMetadataItem( GDAL_DCAP_VECTOR, "YES" );
+    poDriver->SetMetadataItem( GDAL_DCAP_CREATE_LAYER, "YES" );
+    poDriver->SetMetadataItem( GDAL_DCAP_DELETE_LAYER, "YES" );
+    poDriver->SetMetadataItem( GDAL_DCAP_CREATE_FIELD, "YES" );
```

Comment:
MYSQL supports Z and M geometries  (I'd expect it to also support curve geometries, but that should be dealt in a separate pull request by adding OLCMeasuredGeometries and testing. (if OLCMeasuredGeometries isn't declared, the CreateFeature() logic silently linearizes geometries))